diesel_cli/src/pretty_printing.rs

```diff
@@ -0,0 +1,194 @@
+
+
+/// Simple pretty printer hand tailored for the output generated
+/// by the `quote` - crate for schema inference
+///
+/// Rules:
+///   1. Seeing { increases indentation level
+///   2. Seeing } decreases indentation level
+///   3. Insert newline after {, }, ,, and ;
+///   4. Don't put spaces:
+///      between ident and !,
+///      between path segments and ::
+///      after ( and before )
+///      before ,
+pub fn format_schema<W>(schema: &str, mut output: W) where W: ::std::io::Write {
```

Comment:
I think it'd look nicer if you `use std::io::Write` at the top of the file and use `<W: Write>` here instead of a where clause with a fully qualified path.